docs/en/Running-Alluxio-On-Docker.md

```diff
@@ -141,12 +141,32 @@ $ cd opt/alluxio
 $ bin/alluxio runTests
 ```
 
-### Sharing ramdisk with clients
+### Read/Write to the local worker 
 
+There are two options to read/write to the local worker efficiently. The option that shares ramdisk
+with the client requires the memory limit of the client container to be higher than the amount of
+data it writes to the local ramdisk. The domain socket approach does not have this
+limitation but it consumes more CPU.
+
+#### Share ramdisk with clients
 Running the worker with `-v /mnt/ramdisk:/mnt/ramdisk` shares the ramdisk between the worker
```

Comment:
I think it would be nice to distinguish and explain which path is the container path, and which is the host path. Maybe the container path can be `/opt/ramdisk/` or something.